cluster/plan.go

```diff
@@ -1096,3 +1108,14 @@ func appendArgs(command []string, args map[string]string) []string {
 	}
 	return command
 }
+
+func (c *Cluster) IsCRIDockerdEnabled() bool {
+	if c == nil {
+		return false
+	}
+	if c.EnableCRIDockerd != nil && *c.EnableCRIDockerd {
+		logrus.Infof("CRIDockerd is enabled")
```

Comment:
could we move this on cluster level log? since we call this from kubelet process, it'll get printed in info for every call to getPlan per node which'll be pretty often